charts/caasperli/templates/ingress.yaml

```diff
@@ -23,9 +12,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
shouldnt this be `if .Values.ingress.className` to keep it optional?

right now this would render the className as an empty string in all cases